docs/metrics/confusion-matrix.md

```diff
@@ -0,0 +1,204 @@
+| **Actual `Airplane`** | <span class="mg-cell-color-positive">True Negative</span> | <span class="mg-cell-color-negative">False Positive</span> | <span class="mg-cell-color-positive">True Negative</span> |
+| **Actual `Boat`** | <span class="mg-cell-color-negative">False Negative</span> | <span class="mg-cell-color-positive">True Positive</span> | <span class="mg-cell-color-negative">False Negative</span> |
+| **Actual `Car`** | <span class="mg-cell-color-positive">True Negative</span> | <span class="mg-cell-color-negative">False Positive</span> | <span class="mg-cell-color-positive">True Negative</span> |
+
+
+**Example: Multiclass**
+
+Let's take a look at a multiclass classification example and plot a confusion matrix. In this example, we have three
+classes: `Airplane`, `Boat`, and `Car`. The multiclass classifier outputs the following inferences:
+
+<center>
+
+|  | Sample 1 | Sample 2 | Sample 3 | Sample 4 | Sample 5 | Sample 6 |
+| --- | --- | --- | --- | --- | --- | --- |
+| **Ground Truth** | `Airplane` | `Boat` | `Car` | `Airplane` | `Boat` | `Boat` |
+| **Inference** | `Airplane` | `Boat` | `Airplane` | `Airplane` | `Boat` | `Car` |
+
+</center>
+
+A confusion matrix for this example can be plotted:
+
+<center>
+
+|  | Predicted `Airplane` | Predicted `Boat` | Predicted `Car` |
+| --- | --- | --- | --- |
+| `Airplane` | 2 | 0 | 0 |
+| `Boat` | 0 | 2 | 1 |
+| `Car` | 1 | 0 | 0 |
+
+</center>
+
+In a different case, these counts may be much higher:
+
+<center>
+
+|  | Predicted `Airplane` | Predicted `Boat` | Predicted `Car` |
+| --- | --- | --- | --- |
+| `Airplane` | 200 | 0 | 0 |
+| `Boat` | 100 | 8,800 | 600 |
+| `Car` | 100 | 0 | 1,000 |
+
+</center>
+
+This confusion matrix reveals that a model is very good at identifying the `Boat` class: 8,800 of 9,500 `Boat`s were correctly
+predicted. Of the 700 incorrect `Boat` predictions, 600 were predicted as `Car`, and 100 were predicted as `Airplane`.
+
+This confusion matrix indicates that when a model makes a `Airplane` inference, the model is correct half the time. If
+it is incorrect, it has labeled the `Airplane` as a `Boat` or a `Car`.
+
+Whenever there is an actual `Airplane` class, the model never predicts that there is a different transportation object.
+
+### Normalization
+
+Sometimes it is easier to focus on **class-level behavior** if you are using a normalized confusion matrix. If confusion
+matrices are color-coded, normalizing can also create a better visual representation:
+
+![Example of normalized and colored confusion matrix](../assets/images/metrics-confusion-matrix-normalized-light.png#only-light)
+![Example of normalized and colored confusion matrix](../assets/images/metrics-confusion-matrix-normalized-dark.png#only-dark)
+
+You can normalize a confusion matrix by `row` (actual classes), `column` (predicted classes), or `all` (entire matrix).
+Each type of normalization surfaces a view sharing different information, which is outlined below.
```

Comment:
Is it worth linking normalization by row / column to recall / precision?

Reply:
I like that. I added a line that describes the relationship between the row / column normalization to recall / precision.